### PR TITLE
Added lint script in package.json

### LINKED ISSUE #9: Add lint script to package.json
Make a script in `package.json` to run linter instead of needing to type the entire `./node_modules/...` stuff into the terminal.

```
scripts: {
    ...,
    "lint": "eslint .",
},
```

Something like that should work. Run `npm run lint` in your terminal to make sure it works after the changes.

### PR DESCRIPTION
**Pull request by sengfung27**
Closes #9.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "extends": "airbnb"
+}
```
Comment (AryanJ-NYC) [inline .eslintrc.json]: Newline here, please.

File: package.json [hunk 2]
```diff
@@ -6,21 +6,24 @@
   "scripts": {
     "start": " cd views && npm start",
     "build": "cd views && npm run build",
-    "test": "cd views && npm run test "
+    "test": "cd views && npm run test ",
+    "lint": "./node_modules/.bin/eslint app.js; exit 0"
```
Comment (AryanJ-NYC) [inline package.json]: * No need for `./node_modules/.bin/eslint`. `eslint` is enough
* Replace `app.js` with `.`
* Remove `; exit 0`

File: views/error.jade [hunk 3]
```diff
@@ -0,0 +1,6 @@
+extends layout
+
+block content
+  h1= message
+  h2= error.status
+  pre #{error.stack}
```
Comment (AryanJ-NYC) [inline views/error.jade]: What is this file?

File: views/index.jade [hunk 4]
```diff
@@ -0,0 +1,5 @@
+extends layout
+
+block content
+  h1= title
+  p Welcome to #{title}
```
Comment (AryanJ-NYC) [inline views/index.jade]: What is this file?

File: views/layout.jade [hunk 5]
```diff
@@ -0,0 +1,7 @@
+doctype html
+html
+  head
+    title= title
+    link(rel='stylesheet', href='/stylesheets/style.css')
+  body
+    block content
```
Comment (AryanJ-NYC) [inline views/layout.jade]: What is this file?

**Review by AryanJ-NYC — CHANGES_REQUESTED** [review]

**Comment by AryanJ-NYC** [comment]
@sengfung27 Changes to be made.

Add `Closes #9` to the top-level comment (the description) of this pull request .

File: README.md [hunk 6]
```diff
@@ -35,14 +35,19 @@ cd Actively/views/
...
-...
-./node_modules/.bin/eslint File.js
-...
+
+### Setup the Linter
+```
+npm install eslint --save-dev
+./node_modules/.bin/eslint --init
+(Select Popular Style -> AirBnB -> JSON)
+```
```
Comment (AryanJ-NYC) [inline README.md]: The linter is set up. The `Setup the Linter` documentation block is unnecessary. Setting up the linter with a different code style will just cost time in long run. Delete this documentation block.

File: README.md [hunk 7]
```diff
@@ -35,14 +35,19 @@ cd Actively/views/
...
+Running Linter
+```
+npm run lint
+```
+To run on specify file, go to package.json and change it to:
+```
+"lint": "./node_modules/.bin/eslint name_of_the_file.js; exit 0"
+```
+
```
Comment (AryanJ-NYC) [inline README.md]: The linter is run against all files. Delete the specific file documentation block.

File: README.md [hunk 8]
```diff
@@ -35,14 +35,19 @@ cd Actively/views/
...
-...
+
+### Setup the Linter
+```
+npm install eslint --save-dev
+./node_modules/.bin/eslint --init
+(Select Popular Style -> AirBnB -> JSON)
+```
+Running Linter
+```
```
Comment (AryanJ-NYC) [inline README.md]: Use an h3 here (###) for the words `Running Linter`.

**Review by AryanJ-NYC — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (sengfung27) [reply package.json]: I need exit 0, to ignore errors

Comment (AryanJ-NYC) [reply package.json]: Thank you for the explanation. No need to remove `; exit 0` then.

Comment (sengfung27) [reply package.json]: no problem

File: package.json [hunk 9]
```diff
@@ -6,21 +6,24 @@
   "scripts": {
     "start": " cd views && npm start",
     "build": "cd views && npm run build",
-    "test": "cd views && npm run test "
+    "test": "cd views && npm run test ",
+    "lint": "eslint *; exit 0"
   },
```
Comment (AryanJ-NYC) [inline package.json]: @sengfung27 Use a `.`, not a `*`. `*` picks up on `.json` and `.md` files which we don't want to lint.

**Review by AryanJ-NYC — CHANGES_REQUESTED** [review]

File: package.json (hunk 9 above)
Comment (sengfung27) [reply package.json]: got it

**Review by AryanJ-NYC — APPROVED** [review]

**Pull request merged**